scripts/provisioner.sh

```diff
@@ -47,7 +47,17 @@ do
   fi
 done
 set -e
-sleep 10
+
+# If asked to, wait for the total number of control-plane nodes to be registered.
+set +e
```

Comment:
`set +e` is set on line 37 anyways, just remove `set -e` on line 49 and this line and it will have the same outcome